main.tf

```diff
@@ -419,8 +469,7 @@ resource "aws_cloudfront_distribution" "default" {
 
   depends_on = [
     aws_s3_bucket.origin,
-    aws_s3_bucket_ownership_controls.origin,
-    time_sleep.wait_for_aws_s3_bucket_settings
```

Comment:
Is there a reason to drop the time_sleep? Even if OAC doesn't require this, others might still be using OAI

Reply:
It was necessary to drop `time_sleep` because it created circular dependencies